src/hdf5_time_series_storage.jl

```diff
@@ -182,43 +200,78 @@ function remove_time_series!(
     end
 end
 
+function _make_time_array(timestamps, data, columns::Union{Vector{Symbol}, Nothing})
```

Comment:
For an internal function like this I suggest leaving `columns` untyped.

Reply:
Do we need to keep this functionality? It seems this is an out dated thing

Reply:
If the user sets the column name in the time array then this allows that preference to be preserved.

Reply:
Actually, TimeDataContainer now only stores initial times and arrays of data. We will have to document that column names are not preserved in DataFrames and TimeArrays.